shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -434,7 +437,90 @@ private int getPointerDeviceTypeForToolType(int toolType) {
         }
     }
 
-    private void addPointerForIndex(MotionEvent event, int pointerIndex, int pointerChange,
+    private List<PointerAction> expandActions(MotionEvent event, int pointerIndex, int pointerChange,
+    int batched) {
```

Comment:
this line break is odd. I believe you'd either need to indent this further or not line break.